pkgs/development/python-modules/oscpy/default.nix

```diff
@@ -11,6 +11,14 @@ buildPythonPackage rec {
     hash = "sha256-Luj36JLgU9xbBMydeobyf98U5zs5VwWQOPGV7TPXQwA=";
   };
 
+  patches = [
+    # Fix flaky tests with kivy/oscpy#67 - https://github.com/kivy/oscpy/pull/67
+    (fetchpatch {
+      url = "https://github.com/kivy/oscpy/compare/2bc114a97692aef28f8b84d52d0d5a41554a7d93~1..e75ad7f5939cb8759372f7f14f6effd5c5443ccc.diff";
```

Comment:
Can you fetch the commits individually?

Reply:
I've updated it to fetch the 3 commits individually now. Thank you for reviewing!

Reply:
While you're at it please give the patches names using the `name = ` argument.

... actually, looking more closely at that PR, I think we probably only need the first commit. We don't really care about suppressing warnings.

Reply:
I've reduced it to only fetching the one commit and named the patch file. I wasn't sure what an appropriate name would be, so I named it from the commit message. Let me know if that should be changed